src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -61,3 +66,85 @@ def get(self, request: Request, team) -> Response:
             current_day += timedelta(days=1)
 
         return Response(counts)
+
+
+class TriggeredAlertRuleSerializer(AlertRuleSerializer):
+    WEEKS_TO_FETCH = 8
```

Comment:
we might have to get this from the query parameter?

Reply:
Right, I forgot that this is configurable

Reply:
might also be okay to merge this and follow up with that